Reemit.Decompiler.Clr.UnitTests/Metadata/TableReferenceResolverTests.cs

```diff
@@ -0,0 +1,146 @@
+using Reemit.Decompiler.Clr.Metadata;
+using Reemit.Decompiler.Clr.Metadata.Tables;
+
+namespace Reemit.Decompiler.Clr.UnitTests.Metadata;
+
+public class TableReferenceResolverTests
+{
+    [Fact]
+    public void GetReferencedRows_RecordReferencingContiguousRunOfRecordsUntilNextRowRun_ReturnsRunOfRecords()
+    {
+        // Arrange
+        ReferencingRecord[] referencingRecords =
```

Comment:
A bit of a nitpick as this is test code, but we could probably get rid of some duplication here. Not sure if we want to start moving some of this stuff to fixtures, but I've considered it with my tests.

Reply:
A helper method did the job here. But it's probably a good idea to consider using fixtures when there's too much dependency setup/data initialisation going on.